docker/entrypoint-celery-cron-worker.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+exec celery --workdir=/home/app/vaas \
```

Comment:
I wonder, if we will use schedule tasks (cron jobs) why we don't use celery beat for that?

Reply:
we use celery beat, and we use dedicated worker for tasks executed by scheduler. earlier this task was on our default queue ('worker_queue').

Reply:
OK. Thanks for explanation.